### PR TITLE
Issue 35 implement authen admin

### PR DESCRIPTION
**Pull request by minhnguyennn**
## Issue
- Connect #35 

## ToDo
- [x] Task No.1 : Authen sign-in/out with by username
- [x] Task No.2 : Authen reset passsword with by email

### Coding Style
- [x] Check rubocop

### Specs Checking
- [x] Run rspec

## Test Result
[Screencast from 11-12-2023 09:30:12.webm](https://github.com/HopeeSolutionsRuby/demo-app/assets/93121654/74f6435d-edca-4bbd-b710-501773c1f94b)

# ** Note
Run Step : 
1. rails db:create
2. rails db:schema:load
3. rails s

### CODE REVIEW

File: app/assets/stylesheets/administrator/forgot_password.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  .h-custom {
+    height: 100%;
+  }
+}
+
+.link {
+  margin-left: 2px;
+}
+
+.fullVH {
```
Comment (KhoaNDAHopee) [inline app/assets/stylesheets/administrator/forgot_password.scss]: Follow naming conventions https://courses.cs.washington.edu/courses/cse154/17au/styleguide/html-css/naming-conventions-html.html, check another place

File: app/views/administrator/unlocks/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<h2>Resend unlock instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: unlock_path(resource_name), html: { method: :post }) do |f| %>
+<<<<<<< HEAD
```
Comment (KhoaNDAHopee) [inline app/views/administrator/unlocks/new.html.erb]: What is this?

File: app/views/administrator/unlocks/new.html.erb
Comment (KhoaNDAHopee) [inline app/views/administrator/unlocks/new.html.erb:1]: Remove unused views, check another place

File: config/environments/development.rb
Comment (KhoaNDAHopee) [inline config/environments/development.rb:1]: Don't use this config for development

File: app/controllers/administrator/passwords_controller.rb
Comment (KhoaNDAHopee) [inline app/controllers/administrator/passwords_controller.rb:1]: Remove unused controller, check another.

File: app/models/admin.rb
Comment (KhoaNDAHopee) [inline app/models/admin.rb:1]: How about unit test?

**Review by KhoaNDAHopee — CHANGES_REQUESTED** [review]

**Review by khoadev2512 — APPROVED** [review]

**Review by KhoaNDAHopee — APPROVED** [review]

**Pull request merged**